ci-operator/step-registry/cucushift/installer/rehearse/nutanix/ipi/proxy/provision/cucushift-installer-rehearse-nutanix-ipi-proxy-provision-chain.yaml

```diff
@@ -0,0 +1,20 @@
+chain:
+  as: cucushift-installer-rehearse-nutanix-ipi-proxy-provision
+  steps:
+  - chain: ipi-conf-nutanix
+  - ref: ipi-install-rbac
+  - chain: nutanix-provision-bastionhost
+  - ref: mirror-images-by-oc-adm-in-bastion
+  - ref: ipi-conf-proxy
+  - ref: openshift-cluster-bot-rbac
+  - ref: ipi-install-hosted-loki
+  - ref: ipi-install-install
+  - ref: ipi-install-times-collection
```

Comment:
I suggest to use "chain: ipi-install" here.

Reply:
If possible, also pls help update cucushift-installer-rehearse-nutanix-ipi-disconnected-provision together.

Reply:
We need to run "- ref: ipi-install-rbac" before mirror steps, that's the reason why I split "chain: ipi-install", I'll check if install behind proxy still need it.
``` 
 - chain: nutanix-provision-bastionhost
  - ref: mirror-images-by-oc-adm-in-bastion
  `` `

Reply:
ah, got it.

Reply:
install behind proxy do not need run `- ref: ipi-install-rbac` beforehand, updated code